organizers/2018-06-25.html

```diff
@@ -0,0 +1,140 @@
+    <img data-src="/img/thread.png" style="width:10%" alt="Thread" class="plain"></img>
+    <p>
+        <em>We help people to feel happier and more self-confident, by making it easy to dress well, using stylists and data science. Dedicated to maintaining a great working environment, minimising politics and maximising impact.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/timeout.png" style="width:40%" alt="Time Out" class="plain"></img>
+    <p>
+        <em>The trusted global platform that inspires and enables people to experience the best of the city. Operating in 107 cities, across 39 countries, it has an average monthly global audience reach of over 111 million across all platforms.</em>
+    </p>
+</section>
+
+<section style="text-align: left">
+    <h1>📋 Today's Schedule</h1>
+    <h2>🎺 Django News</h2>
+    <h2>🎺 Python News</h2>
+    <h2>🎺 Developers News</h2>
+</section>
+
+<section style="text-align: left">
+    <h1 style="text-align: center;">📋 Today's Schedule</h1>
+    <h2 style="text-align: center;">Talks</h2>
+    <ul>
+        <li>
+            Matthew Hall - 10 Lessons From 10 Years of Selling Dev Work
+        </li>
+        <li>
+            Lili Kastilio - pip install disaster-waiting-to-happen
+        </li>
+    </ul>
+</section>
+
+<section style="text-align: left">
+    <h1>📋 Today's Schedule</h1>
+    <h2>Eating and Drinking</h2>
+    <ol>
+        <li><strong>Until ~20:45</strong> - Drinks and Pizza at<br>The Space Bar</li>
+        <li><strong>20:45 - close</strong>: Red Lion Pub (opposite)</li>
+    </ol>
+</section>
+
+<section>
+    <h1>🎺 Django News</h1>
+    <h2>Bugfix release</h2>
+    <ul>
+        <li>2.0.6 bugfix release</li>
+        <li>2.1 beta release</li>
+	    <li><a href="https://www.djangoproject.com/weblog/">https://www.djangoproject.com/weblog/</a></li>
+    </ul>
+</section>
+
+<section>
+    <h1>🎺 Django News</h1>
+    <h2>DjangoCon Europe 2018</h2>
+    <ul>
+        <li>May 23-27, 2018 · Heidelberg, Germany</li>
+        <li>Talks available on <a href="https://www.youtube.com/playlist?list=PLY_che_OEsX3aZo5RttI6Fj2XZ7nTjhBu">YouTube</a></li>
+    </ul>
+
+    <h2 style="margin-top: 0.4em;">DJUGL</h2>
+    <ul>
+        <li>July 17, 2018</li>
+        <li>Registration on <a href="https://www.eventbrite.co.uk/e/djugl-django-user-group-london-tickets-46930607711">eventbrite</a></li>
+    </ul>
+</section>
+
+<section>
+    <h1>🎺 Python News</h1>
+    <h2>Euro Python</h2>
+    <ul>
+        <li>July 23-29, 2018 · Edinburgh, Scotland, UK</li>
+        <li>Tickets on sale at the <a href="https://ep2018.europython.eu/en/">EuroPython Website</a></li>
+    </ul>
+</section>
+
+<section>
+    <h1>🎺 Developers News</h1>
+    <h2>International Women in Engineering Day</h2>
+    <ul>
+        <li>June 23, 2018</li>
+        <li>Events still happening, list at <a href="http://www.inwed.org.uk/">INWED officil website</a></li>
+    </ul>
+</section>
```

Comment:
You can maybe add the London Python sprint that happened last week. It was a documentation sprint and Django was one of the projects that was proposed:
    * pandas
    * matplotlib
    * jupyter
    * Django
    * numpy

https://www.meetup.com/Python-Sprints/events/251714652/

Organizers asked me to announce that London Django Meetup attendees are more then welcome to join upcoming sprints.

Reply:
I went to that sprint, but I don't think it's useful for us to talk about *past* events so much as future ones